src/licensedcode/data/licenses/atmosphere.yml

```diff
@@ -0,0 +1,6 @@
+key: atmosphere
```

Comment:
This has a version "Atmosphere Software License Version 0.4" so please use instead `atmosphere-0.4` as a key